web/src/rest/index.js

```diff
@@ -0,0 +1,13 @@
+import girderRest from '@/rest/girder';
+import publishRest from '@/rest/publish';
+import toggles from '@/featureToggle';
+
+const loggedIn = () => ((toggles.UNIFIED_API) ? !!publishRest.user : !!girderRest.user);
+const user = () => ((toggles.UNIFIED_API) ? publishRest.user : girderRest.user);
```

Comment:
Not something you changed in this PR, but I just realized this could be made a tiny bit simpler:
```suggestion
const user = () => ((toggles.UNIFIED_API) ? publishRest.user : girderRest.user);
const loggedIn = () => !!user();
```